### PR TITLE
Change wording for the sponsor page

### LINKED ISSUE #488: Phrasing on sponsor page
> Last year, JuliaCon was held at MIT in Boston, USA. This year, we are thrilled to be returning in-person to Europe.

This reads awkwardly - implying that being back in Europe is better than being in Boston. 

I think the ‘thrill’ was returning to in-person (after the pandemic).

### PR DESCRIPTION
**Pull request by bvdmitri**
This PR fixes #488

### CODE REVIEW

File: 2024/sponsor.md [hunk 1]
```diff
@@ -4,11 +4,10 @@
 JuliaCon is a community organized non-profit conference, and it would not be possible without the support of its amazing set of sponsors over the years.
 We invite your organization to be a sponsor of JuliaCon 2024. You can find our [sponsors from last year](https://juliacon.org/2023/) at the bottom of our 2023 homepage.
 
-JuliaCon is the largest annual conference organized around Julia, the fastest high-performance open source computing language for machine learning, artificial intelligence, life sciences, robotics, algorithmic trading, real-time analytics and more.
+JuliaCon is the largest annual conference organized around Julia, the fastest high-performance open source computing language for machine learning, artificial intelligence, life sciences, robotics, algorithmic trading, real-time analytics and more. This year the conference is set to take place at the iconic [PSV Stadium in Eindhoven](http://localhost:8000/2024/venue/), the Netherlands, July 9th - 12th, 2024.
```
Comment (giordano) [inline 2024/sponsor.md:7]: Would something like
```suggestion
JuliaCon is the largest annual conference organized around Julia, the fastest high-performance open source computing language for machine learning, artificial intelligence, life sciences, robotics, algorithmic trading, real-time analytics and more. This year the conference is set to take place at the iconic [PSV Stadium in Eindhoven](/2024/venue/), the Netherlands, July 9th - 12th, 2024. 
```
work?  In any case not localhost 😛

File: 2024/sponsor.md [hunk 2]
```diff
@@ -4,11 +4,10 @@
...
 
-JuliaCon is the largest annual conference organized around Julia, the fastest high-performance open source computing language for machine learning, artificial intelligence, life sciences, robotics, algorithmic trading, real-time analytics and more.
+JuliaCon is the largest annual conference organized around Julia, the fastest high-performance open source computing language for machine learning, artificial intelligence, life sciences, robotics, algorithmic trading, real-time analytics and more. This year the conference is set to take place at the iconic [PSV Stadium in Eindhoven](http://localhost:8000/2024/venue/), the Netherlands, July 9th - 12th, 2024. 
 
-Last year, JuliaCon was held at MIT in Boston, USA. This year, we are thrilled to be returning in-person to Europe. With a global audience of developer, JuliaCon offers you the unique opportunity to present your company and your product to a huge group of engaged scientific programmers.
+With a global audience of developer, JuliaCon offers you the unique opportunity to present your company and your product to a huge group of engaged scientific programmers.
```
Comment (giordano) [inline 2024/sponsor.md:9]: Trailing whitespace
```suggestion
With a global audience of developer, JuliaCon offers you the unique opportunity to present your company and your product to a huge group of engaged scientific programmers.
```

File: 2024/sponsor.md (hunk 1 above)
Comment (bvdmitri) [reply 2024/sponsor.md:7]: ahah, nice catch!

Comment (bvdmitri) [reply 2024/sponsor.md]: yes, it works!

**Pull request merged**